build.cake

```diff
@@ -109,15 +107,15 @@ Task("Build")
     .IsDependentOn("NuGetRestore")
     .Does(() =>
     {
-        DotNetCoreBuild(SOLUTION_FILE, CreateDotNetCoreBuildSettings());
+        DotNetBuild(SOLUTION_FILE, CreateDotNetCoreBuildSettings());
     });
 
-DotNetCoreBuildSettings CreateDotNetCoreBuildSettings() =>
-    new DotNetCoreBuildSettings
+DotNetBuildSettings CreateDotNetCoreBuildSettings() =>
+    new DotNetBuildSettings
     {
         Configuration = configuration,
         NoRestore = true,
-        Verbosity = DotNetCoreVerbosity.Minimal
+        Verbosity = DotNetVerbosity.Minimal
```

Comment:
I had added the below to ensure the version is part of the dll.
```cake
MSBuildSettings = new DotNetMSBuildSettings
{
    Version = packageVersion
}
```

This was previously done on AppVeyor builds only, but that was removed.

Reply:
Done

Reply:
But it has a property of that type, so moved there